.release-please-manifest.json

```diff
@@ -1,134 +1 @@
.7.0","java-bigqueryreservation":"2.4.5","java-billingbudgets":"2.3.5","java-billing":"2.4.0","java-binary-authorization":"1.2.8","java-certificate-manager":"0.5.3","java-channel":"3.7.5","java-cloudbuild":"3.5.5","java-cloudcommerceconsumerprocurement":"0.1.3","java-compute":"1.13.0","java-contact-center-insights":"2.4.0","java-containeranalysis":"2.4.8","java-container":"2.6.0","java-datacatalog":"1.9.5","java-dataflow":"0.7.5","java-dataform":"0.2.3","java-data-fusion":"1.3.4","java-datalabeling":"0.123.6","java-dataplex":"1.2.0","java-dataproc-metastore":"2.5.0","java-dataproc":"4.1.0","java-datastream":"1.3.0","java-debugger-client":"1.3.5","java-deploy":"1.1.7","java-dialogflow-cx":"0.15.0","java-dialogflow":"4.9.0","java-dlp":"3.8.0","java-dms":"2.2.6","java-dns":"2.1.8","java-document-ai":"2.8.0","java-domains":"1.0.5","java-errorreporting":"0.125.0-beta","java-essential-contacts":"2.3.5","java-eventarc":"1.3.7","java-eventarc-publishing":"0.3.5","java-filestore":"1.4.4","java-functions":"2.5.3","java-game-servers":"2.4.0","java-gke-backup":"0.2.5","java-gke-connect-gateway":"0.4.6","java-gkehub":"1.3.5","java-gke-multi-cloud":"0.2.5","java-grafeas":"2.4.6","java-gsuite-addons":"2.3.5","java-iam-admin":"2.0.0","java-iamcredentials":"2.3.5","java-ids":"1.2.5","java-iot":"2.3.5","java-kms":"2.6.8","java-language":"2.5.0","java-life-sciences":"0.5.5","java-managed-identities":"1.2.0","java-mediatranslation":"0.9.5","java-memcache":"2.3.5","java-monitoring-dashboards":"2.5.5","java-monitoring":"3.4.6","java-networkconnectivity":"1.2.6","java-network-management":"1.5.0","java-network-security":"0.6.3","java-notebooks":"1.2.0","java-notification":"0.123.15-beta","java-optimization":"1.2.0","java-orchestration-airflow":"1.3.5","java-orgpolicy":"2.3.5","java-os-config":"2.5.5","java-os-login":"2.2.6","java-phishingprotection":"0.34.5","java-policy-troubleshooter":"1.2.5","java-private-catalog":"0.6.0","java-profiler":"2.4.0","java-recaptchaenterprise":"3.1.0","java-recommendations-ai":"0.10.4","java-recommender":"2.5.5","java-redis":"2.6.5","java-resourcemanager":"1.5.6","java-resource-settings":"1.3.4","java-retail":"2.6.0","java-run":"0.4.0","java-scheduler":"2.3.12","java-secretmanager":"2.4.0","java-securitycenter":"2.11.0","java-securitycenter-settings":"0.7.0","java-security-private-ca":"2.6.0","java-service-control":"1.4.0","java-servicedirectory":"2.5.0","java-service-management":"3.2.0","java-service-usage":"2.4.0","java-shell":"2.2.7","java-speech":"3.0.0","java-storage-transfer":"1.4.0","java-talent":"2.4.5","java-tasks":"2.3.11","java-texttospeech":"2.4.5","java-tpu":"2.4.0","java-trace":"2.4.0","java-translate":"2.3.5","java-video-intelligence":"2.3.0","java-video-live-stream":"0.5.5","java-video-stitcher":"0.3.5","java-video-transcoder":"1.3.0","java-vision":"3.2.0","java-vmmigration":"1.4.0","java-vpcaccess":"2.5.0","java-webrisk":"2.2.5","java-websecurityscanner":"2.3.0","java-workflow-executions":"2.3.5","java-workflows":"2.3.5"}
```

Comment:
What happened here to the line breaks?

Reply:
This update was done by release-please in the release:main branch only. I think there is some indentation issue we need to fix. As such, since this is valid json, nothing is incorrect in this, only need to make it human readable. 
Will have to include some spaces in the config generating logic (as corrected by Jeff) see below image:
![image](https://user-images.githubusercontent.com/90280028/193640342-157f846d-e9b0-4e1a-ab5c-d11868402bdb.png)